src/core/ArxGame.cpp

```diff
@@ -658,6 +664,10 @@ static bool HandleGameFlowTransitions() {
 		LoadLevelScreen(g_areaToLoad);
 		
 		DanaeLoadLevel(g_areaToLoad);
+		
+		USE_PLAYERCOLLISIONS = g_initialPlayerCollision;
+		g_initialPlayerCollision = true;
```

Comment:
why is it set back to true here? wouldnt that mean noclicp gets deactivated on the next level transition?

Reply:
Yes, it means exactly that. Otherwise every time the player changes level noclip would get re-enabled even if the player turns it off.

Reply:
`true` is the default value for USE_PLAYERCOLLISIONS
![image](https://github.com/arx-insanity/ArxLibertatis/assets/2386064/8cfff978-1e13-470d-ba5c-224c494a6491)